src/python/grpcio_tests/tests_py3_only/interop/xds_interop_client.py

```diff
@@ -54,6 +54,11 @@
     "EmptyCall": messages_pb2.ClientConfigureRequest.EMPTY_CALL,
 }
 
+_METHOD_CAMEL_TO_FULL_PATH = {
```

Comment:
`_METHOD_CAMEL_TO_CAPS_SNAKE` above can be removed.

Reply:
Hadn't gotten to clean-up yet.